src/mdb_v8.c

```diff
@@ -706,6 +721,11 @@ autoconfigure(v8_cfg_t *cfgp)
 		failed++;
 	}
 
+	if (V8_TYPE_JSTYPEDARRAY == -1) {
```

Comment:
Is it definitely a problem for there to be no JSTypedArray type?  Even on older versions of Node?